2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -24,25 +24,58 @@ const myBooks = [
     author: 'Don Norman',
     isbn: '978-0465050659',
     alreadyRead: false,
+    coverURL: 'https://miro.medium.com/max/1049/0*CmkOVjGAnm3D0rTo',
   },
   {
     title: 'The Most Human Human',
     author: 'Brian Christian',
     isbn: '978-1617933431',
     alreadyRead: true,
+    coverURL:
+      'https://www.kurzweilai.net/images/The-Most-Human-Human-Paperback-Front-Cover.jpg',
   },
   {
     title: 'The Pragmatic Programmer',
     author: 'Andrew Hunt',
     isbn: '978-0201616224',
     alreadyRead: true,
+    coverURL:
+      'https://static.epub.vn/epubvn-image/optimized-cover1565689052299-220px-the_pragmatic_programmer.jpg',
   },
 ];
 
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
-}
+  const listULTemp = document.createElement('ul');
+  listULTemp.setAttribute('style', 'display: flex; flex-direction: row;');
+  books.forEach((bookObject) => {
+    const nameAuthor = document.createElement('p');
+    const textNode = document.createTextNode(
+      `${bookObject.title} - ${bookObject.author}`
+    );
+    nameAuthor.appendChild(textNode);
+    const bookImageURL = document.createElement('img');
+    bookImageURL.setAttribute('src', `${bookObject.coverURL}`);
```

Comment:
```suggestion
    bookImageURL.setAttribute('src', bookObject.coverURL);
```

Template strings are only needed when combining variables and string together